### PR TITLE
Change styling of UI and code

### PR DESCRIPTION
**Pull request by moziliar**

### CODE REVIEW

File: src/util/const.py [hunk 1]
```diff
@@ -9,32 +9,42 @@
...
-    f'{ADD_FAVORITE_DESC}{REMOVE_FAVORITE_DESC}{SET_BREAKFAST_NOTIFICATION_DESC}' \
-    f'{SET_DINNER_NOTIFICATION_DESC}{SETTINGS_DESC}{HELP_DESC}\n' \
-    f'{BREAKFAST_COMMAND} (or {DINNER_COMMAND}) <day> - view the breakfast/dinner menu for a particular day\n' \
-    'e.g. /breakfast tomorrow, /breakfast saturday, /dinner next tuesday\n\n' \
-    'Give feedback for the bot at https://github.com/rc-dining-bot/telegram-bot'
+    f'{BREAKFAST_DESC}' \
+    f'{DINNER_DESC}' \
+    f'{ADD_FAVORITE_DESC}' \
+    f'{REMOVE_FAVORITE_DESC}' \
+    f'{SET_BREAKFAST_NOTIFICATION_DESC}' \
```
Comment (jonchan51) [inline src/util/const.py]: Should we remove this first since we're not implementing?

File: src/util/kb_mark_up.py [hunk 2]
```diff
@@ -1,34 +1,50 @@
...
 
 
 def build_menu(buttons,
                n_cols,
                header_buttons=None,
                footer_buttons=None):
+    # builds the menu with:
+    # array of buttons
+    # number of rows
+    # (optionally) header/footer buttons
```
Comment (jonchan51) [inline src/util/kb_mark_up.py]: mention that it should be an array

File: src/util/messages.py [hunk 3]
```diff
@@ -26,24 +26,24 @@ def failed_to_parse_date_msg(entered_date):
 
 # setting messages
 def settings_msg():
-    return 'Hi, here is the menu for settings:\n'
+    return '<b>Hi, here is the menu for settings:</b>\n'
 
 
 def no_hidden_cuisine_msg():
-    return 'You do not have any hidden cuisine.'
+    return '<b>You do not have any hidden cuisine.</b>'
```
Comment (jonchan51) [inline src/util/messages.py]:
```suggestion
    return '<b>You do not have any hidden cuisines.</b>'
```

**Review by jonchan51 — CHANGES_REQUESTED** [review]
I think one thing we can add are default keyboard buttons for breakfast and dinner for today. so that the user doesn't need to type, can just press the button immediately. If they want other days then need to type

File: src/util/const.py (hunk 1 above)
Comment (moziliar) [reply src/util/const.py]: Yes

File: src/util/kb_mark_up.py (hunk 2 above)
Comment (moziliar) [reply src/util/kb_mark_up.py]: ok

**Comment by moziliar** [comment]
> I think one thing we can add are default keyboard buttons for breakfast and dinner for today. so that the user doesn't need to type, can just press the button immediately. If they want other days then need to type

I think the links there are useful still. Can review the current UI and see if we should have the buttons instead? Could replace the /bfast /dinner with buttons but not sure if that'd enhance readability.

**Review by jonchan51 — APPROVED** [review]

**Pull request merged**